content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -22,11 +22,59 @@ generates log messages for the Kubernetes system components.
 
 For more information about klog configuration, see the [Command line tool reference](/docs/reference/command-line-tools-reference/).
```

Comment:
What is the right approach for updating the command line tools reference? Manually edit the HTML table?

That needs to be part of this PR.

/hold

Reply:
We typically update all [generated](https://github.com/kubernetes-sigs/reference-docs) docs during the release, separately from human-made PRs that accompany changes in that release.

If that's true here to, we can /unhold.

Reply:
Sounds like I don't need to worry about the reference docs then.

/unhold